### PR TITLE
Allow nav bar to scroll above the view controller edges

### PR DESCRIPTION
**Pull request by ManueGE**
Solves the issue in #351: 

<a href="https://imgflip.com/gif/2lahqn"></a>

If you like the idea but you'd like to use better wording or a different approach, let me know, I'll be happy to help.

### CODE REVIEW

File: Source/ScrollingNavigationController.swift [hunk 1]
```diff
@@ -171,9 +174,10 @@ open class ScrollingNavigationController: UINavigationController, UIGestureRecog
    - parameter delay: The delay expressed in points that determines the scrolling resistance. Defaults to `0`
    - parameter scrollSpeedFactor : This factor determines the speed of the scrolling content toward the navigation bar animation
    - parameter collapseDirection : The direction of scrolling that the navigation bar should be collapsed
+   - parameter additionalOffset : The additional distance that the navigation bar can move up after reaching the top of the screen. Defaults to 0
    - parameter followers: An array of `NavigationBarFollower`s that will follow the navbar. The wrapper holds the direction that the view will follow
    */
-  open func followScrollView(_ scrollableView: UIView, delay: Double = 0, scrollSpeedFactor: Double = 1, collapseDirection: NavigationBarCollapseDirection = .scrollDown, followers: [NavigationBarFollower] = []) {
+  open func followScrollView(_ scrollableView: UIView, delay: Double = 0, scrollSpeedFactor: Double = 1, collapseDirection: NavigationBarCollapseDirection = .scrollDown, additionalOffset: CGFloat = 0, followers: [NavigationBarFollower] = []) {
```
[the remark this answers is not in the document]
Comment (ManueGE) [reply Source/ScrollingNavigationController.swift]: Should I fix this? I see that a lot of lines are > 120

File: Source/ScrollingNavigationController.swift [hunk 2]
```diff
@@ -171,9 +174,10 @@ open class ScrollingNavigationController: UINavigationController, UIGestureRecog
    - parameter delay: The delay expressed in points that determines the scrolling resistance. Defaults to `0`
    - parameter scrollSpeedFactor : This factor determines the speed of the scrolling content toward the navigation bar animation
    - parameter collapseDirection : The direction of scrolling that the navigation bar should be collapsed
+   - parameter additionalOffset : The additional distance that the navigation bar can move up after reaching the top of the screen. Defaults to 0
```
[the remark this answers is not in the document]
Comment (ManueGE) [reply Source/ScrollingNavigationController.swift]: Should I fix this? I see that a lot of lines are > 120

**Comment by andreamazz** [comment]
Hey @ManueGE 
thank you for the PR, I'll check it out and merge as soon as possible.
Don't worry about the 120 chars lines, I've been meaning to disable that rule anyway.

**Comment by andreamazz** [comment]
Ok, looking good. Thank you for the contribution!

**Pull request merged**